apps/aklite-offline/cmds.h

```diff
@@ -18,21 +18,28 @@ class Cmd {
  public:
   using Ptr = std::shared_ptr<Cmd>;
   const std::string& name() const { return _name; }
+  const std::string& description() const { return _description; }
   const po::options_description& options() const { return _options; }
 
   virtual int operator()(const po::variables_map& vm) const = 0;
 
  protected:
-  Cmd(std::string name, const po::options_description& options) : _name{std::move(name)}, _options{options} {}
+  Cmd(std::string name, std::string description, const po::options_description& options)
+      : _name{std::move(name)}, _description{std::move(description)}, _options{options} {}
 
  private:
   const std::string _name;
+  const std::string _description;
   const po::options_description& _options;
 };
 
 class CheckCmd : public Cmd {
  public:
-  CheckCmd() : Cmd("check", _options) {
+  CheckCmd()
+      : Cmd("check",
```

Comment:
The `check` command also checks/validates the TUF metadata of the bundle as well as its signature and validity.

Reply:
I'm now mentioning the validation in the description. Tried to keep the description short, so I suppose "validation" implies that the signature and validity are verified.